pkg/rpcclient/actor/waiter.go

```diff
@@ -125,6 +137,11 @@ func (w *PollingWaiter) Wait(h util.Uint256, vub uint32, err error) (*state.AppE
 	if err != nil {
 		return nil, err
 	}
+	return w.WaitOne(context.TODO(), vub, h)
```

Comment:
Use client's context?

Reply:
It already uses it under the hood (`RPCPollingWaiter` has `Context()` method which returns client's context). This `context.TODO` is additional context not related to the client, so `WaitAny(ctx context.Context, ...)` handles Client's context by default and the second context passed as an argument, like it was described [here](https://github.com/nspcc-dev/neo-go/pull/2749#discussion_r1000539783).

Reply:
Yeah, I know, it technically is correct, maybe I just don't like `context.TODO()` and would prefer reusing client's context yet another time. But it's OK to leave it as is..